src/js/gi/selectors/calculator.js

```diff
@@ -0,0 +1,23 @@
+import { createSelector } from 'reselect';
+
+const getConstants = (state) => state.constants.constants;
+
+const getEligibilityDetails = (state) => {
+  const details = Object.assign({}, state.eligibility);
+  delete details.dropdowns;
```

Comment:
My approach is to try to make inputs as explicit/obvious as possible - like I did for ` getRequiredAttributes` in the `estimator.js` selector.